tests/core/v5_1/alexandria/test_alexandria_client.py

```diff
@@ -131,6 +131,17 @@ async def test_alexandria_client_send_find_nodes(
         assert message.payload.distances == (0, 255, 254)
 
 
+@pytest.mark.trio
```

Comment:
@pipermerriam one final :eyes: for this test. This is the only test I could find that seems to test directly exercise the update to `AlexandriaClient.subscribe_request()`. From what I can tell, `AlexandriaClient` only uses the `subscribe_request()` api in `find_nodes()`.

Reply:
Correct!  didn't realize that the `AlexandriaClient.ping()` implementation goes directly through the `NetworkAPI.talk()`.  :+1: